samples/node/pub_sub_electron_node/preload_pubsub5.ts

```diff
@@ -0,0 +1,201 @@
+    client = new mqtt5.Mqtt5Client(config);
+
+    client.on('error', (error: ICrtError) => {
+        log("Error event: " + error.toString());
+    });
+
+    client.on("messageReceived",(eventData: mqtt5.MessageReceivedEvent) : void => {
+        log("Message Received event: " + JSON.stringify(eventData.message));
+        if (eventData.message.payload) {
+            log("  with payload: " + toUtf8(new Uint8Array(eventData.message.payload as ArrayBuffer)));
+        }
+    } );
+
+    client.on('attemptingConnect', (eventData: mqtt5.AttemptingConnectEvent) => {
+        log("Attempting Connect event");
+    });
+
+    client.on('connectionSuccess', (eventData: mqtt5.ConnectionSuccessEvent) => {
+        log("Connection Success event");
+        log("Connack: " + JSON.stringify(eventData.connack));
+        log("Settings: " + JSON.stringify(eventData.settings));
+    });
+
+    client.on('connectionFailure', (eventData: mqtt5.ConnectionFailureEvent) => {
+        log("Connection failure event: " + eventData.error.toString());
+        if (eventData.connack) {
+            log("Connack: " + JSON.stringify(eventData.connack));
+        }
+    });
+
+    client.on('disconnection', (eventData: mqtt5.DisconnectionEvent) => {
+        log("Disconnection event: " + eventData.error.toString());
+        if (eventData.disconnect !== undefined) {
+            log('Disconnect packet: ' + JSON.stringify(eventData.disconnect));
+        }
+    });
+
+    client.on('stopped', (eventData: mqtt5.StoppedEvent) => {
+        log("Stopped event");
+    });
+
+    return client;
+}
+
+async function runSample(args : any) {
+
+    try{
+
+        let client : mqtt5.Mqtt5Client = createClient(args);
+
+        const connectionSuccess = once(client, "connectionSuccess");
+
+        client.start();
+        await connectionSuccess;
+
+        const suback = await client.subscribe({
+            subscriptions: [
+                { qos: mqtt5.QoS.AtLeastOnce, topicFilter: qos1_topic },
+                { qos: mqtt5.QoS.AtMostOnce, topicFilter: qos0_topic }
+            ]
+        });
+        log('Suback result: ' + JSON.stringify(suback));
+
+        const qos0PublishResult = await client.publish({
+            qos: mqtt5.QoS.AtMostOnce,
+            topicName: qos0_topic,
+            payload: JSON.stringify("This is a qos 0 payload"),
+            userProperties: [
+                {name: "test", value: "userproperty"}
+            ]
+        });
+        log('QoS 0 Publish result: ' + JSON.stringify(qos0PublishResult));
+
+        const qos1PublishResult = await client.publish({
+            qos: mqtt5.QoS.AtLeastOnce,
+            topicName: qos1_topic,
+            payload: JSON.stringify("This is a qos 1 payload")
+        });
+        log('QoS 1 Publish result: ' + JSON.stringify(qos1PublishResult));
+    }
+    catch(error)
+    {
+        log("Client failed: " + error)
+    }
+
+}
+
+export const PubSub5 = async () => {
```

Comment:
**Extremely minor** nitpick: I might rename this function to `PubSub5Connect` so it follows the same format as the other exposed functions and rename the `runSample` function to something like `createClientAndConnect` or something, since it's no longer the entire sample but just the connect part invoked when pressing the connect button.

Admitted though, this is incredibly minor and by no means necessary in the slightest. 😅